pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -130,6 +130,21 @@ def test_engine_kwargs_append_data_only(ext, data_only, expected):
             DataFrame().to_excel(writer, sheet_name="Sheet2")
 
 
+@pytest.mark.parametrize("kwarg_name", ["read_only", "data_only"])
+@pytest.mark.parametrize("kwarg_value", [True, False])
+def test_engine_kwargs_append_reader(datapath, ext, kwarg_name, kwarg_value):
+    # GH 55027
+    # test that `read_only` and `data_only` can be passed to
+    #  `openpyxl.reader.excel.load_workbook` via `engine_kwargs`
+    from pandas.io.excel._openpyxl import OpenpyxlReader
```

Comment:
Could you import this at the top of the file?

Reply:
Sure. Done.